clusters/app.ci/supplemental-ci-images/openshift-priv/admin_roles.yaml

```diff
@@ -32,9 +32,9 @@ items:
     name: image-tagger-ci-operator
     namespace: ocp-private
   roleRef:
+    apiGroup: rbac.authorization.k8s.io
     kind: Role
     name: image-tagger
-    namespace: ocp-private
```

Comment:
why is this removed?

Reply:
```
$ oc apply -f  clusters/app.ci/supplemental-ci-images/openshift-priv/admin_roles.yaml --dry-run=server --as system:admin --validate=true
error: error validating "clusters/app.ci/supplemental-ci-images/openshift-priv/admin_roles.yaml": error validating data: [ValidationError(RoleBinding.roleRef): unknown field "namespace" in io.k8s.api.rbac.v1.RoleRef, ValidationError(RoleBinding.roleRef): missing required field "apiGroup" in io.k8s.api.rbac.v1.RoleRef]; if you choose to ignore these errors, turn validation off with --validate=false
```

I think it is has to be the role in the same namespace as the rolebinding.